lib/castle/client_actions/list_item.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+module Castle
+  module ClientActions
+    module ListItem
+      def create_list(options)
+        Castle::API::ListItem::Create.call(options)
+      end
+
+      def update_list(options)
+        Castle::API::ListItem::Update.call(options)
+      end
+
+      def delete_list(options)
+        Castle::API::ListItem::Delete.call(options)
+      end
+
+      def all_lists(options)
+        Castle::API::ListItem::All.call(options)
+      end
+
+      def get_list(options)
+        Castle::API::ListItem::Get.call(options)
+      end
+
+      def query_lists(options)
+        Castle::API::ListItem::Query.call(options)
+      end
```

Comment:
I believe these should be named differently to don't conflict with the `Lists` actions:
```suggestion
      # @param options [Hash]
      def create_list_item(options)
        Castle::API::ListItems::Create.call(options)
      end

      def update_list_item(options)
        Castle::API::ListItems::Update.call(options)
      end

      def delete_list_item(options)
        Castle::API::ListItems::Delete.call(options)
      end

      def get_all_list_items(options)
        Castle::API::ListItems::GetAll.call(options)
      end

      def get_list_item(options)
        Castle::API::ListItems::Get.call(options)
      end

      def query_lists(options)
        Castle::API::ListItem::Query.call(options)
      end
```

Tests for the `Castle::Client` could have caught that.